lsp-rust.el

```diff
@@ -456,6 +456,12 @@ The command should include `--message=format=json` or similar option."
   :group 'lsp-rust
   :package-version '(lsp-mode . "6.3.2"))
 
+(defcustom lsp-rust-analyzer-inlay-face 'font-lock-comment-face
+  "The face to use for the Rust Analyzer inlays."
+  :type 'face
+  :group 'lsp-rust
+  :package-version '(lsp-mode . "6.3.2"))
```

Comment:
can you use 6.4?

Reply:
9ee929e281375765afda05995badb45a3e523347